test/commands/builds/index.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+/* globals describe beforeEach it */
```

Comment:
I actually copied this test from another suite, and it appears this comment is entirely unnecessary, without any need for other changes. I've removed it.